### PR TITLE
1.9.0

### PR DESCRIPTION
**Pull request by patr1ckm**
Upgrade notebooks to new `datascience-r` image. Previously, we inherited from `rocker/verse` and installed `civis` manually again. Instead, we inherit from `datascience-r`.

[Test Notebook](https://platform.civisanalytics.com/spa/#/notebooks/14449?expanded=true)

### CODE REVIEW

File: setup.R [hunk 1]
```diff
@@ -1,10 +1,6 @@
-# Install civis R client
-options(unzip='internal');
-devtools::install_github('civisanalytics/civis-r', ref = 'v2.1.0', upgrade_dependencies = FALSE);
-
 # Install R Kernel for Jupyter
 install.packages(c('IRdisplay', 'pbdZMQ'))
-devtools::install_github('IRkernel/IRkernel', ref = '1.0.0', upgrade_dependencies = FALSE);
+devtools::install_github('IRkernel/IRkernel', ref = '1.0.2')
```
Comment (patr1ckm) [inline setup.R]: `upgrade_dependencies` is no longer an argument. The same behavior is retained using default arguments.

Upgrading IRkernal from `1.0.0 -> 1.0.2` should improve code completion.

Comment (elsander) [reply setup.R]: Do you not need to install the client itself anymore?

Comment (patr1ckm) [reply setup.R]: Nope, it's coming from `datascience-r`.

Comment (elsander) [reply setup.R]: Ah, of course, got it.

**Comment by elsander** [comment]
Also flagging this change for @Icehawk78 since this release will change the default R notebook image in production.

**Pull request merged**